app/components/article-featured-video.js

```diff
@@ -60,6 +60,10 @@ export default Component.extend(
 			}
 		},
 
+		getMainElement() {
+			return document.getElementsByClassName(this.get('classNames')[0])[0];
```

Comment:
this.$()[0]; instead of `document.getElementsByClassName(this.get('classNames')[0])[0]`